src/engine/skelmodel.h

```diff
@@ -1810,17 +1810,20 @@ template<class MDL> struct skelcommands : modelcommands<MDL, struct MDL::skelmes
 
         part *p = (part *)MDL::loading->parts.last();
 
-        vector<char *> bonestrs;
+        std::vector<std::string> bonestrs;
         explodelist(maskstr, bonestrs);
         vector<ushort> bonemask;
         loopv(bonestrs)
         {
-            char *bonestr = bonestrs[i];
+            char const *bonestr = bonestrs[i].data();
             int bone = p->meshes ? ((meshgroup *)p->meshes)->skel->findbone(bonestr[0]=='!' ? bonestr+1 : bonestr) : -1;
-            if(bone<0) { conoutf("\frcould not find bone %s for anim part mask [%s]", bonestr, maskstr); bonestrs.deletearrays(); return; }
+            if(bone<0)
+            {
+                conoutf("\frcould not find bone %s for anim part mask [%s]", bonestr, maskstr);
+                return;
+            }
             bonemask.add(bone | (bonestr[0]=='!' ? BONEMASK_NOT : 0));
         }
-        bonestrs.deletearrays();
```

Comment:
why did you remove this here instead of replacing it with `bonestr.clear()` like you did in other files too?